src/main/java/com/github/stephengold/joltjni/AaBox.java

```diff
@@ -456,6 +626,19 @@ native private static void expandBy(
 
     native private static float getVolume(long boxVa);
 
+    native private static float getSurfaceArea(long boxVal);
+
+    native private static float[] getSupport(
+            long boxVal, float dx, float dy, float dz);
+
+    native private static float getSqDistanceTo(
+            long boxVal, float px, float py, float pz);
+
+    native private static boolean overlaps(long boxVal, long inOther);
+
+    native private static boolean overlaps(
+            long boxVal, float pc, float pnx, float pny, float pnz);
```

Comment:
overlaps(long, long) should come after overlaps(long, float, ...) because F comes before L in the alphabet